src/tests/homework.spec.js

```diff
@@ -0,0 +1,136 @@
+import { expect, test } from "@playwright/test";
+
+async function openRegistrationPage(page) {
+  await page.goto("/registrace");
+}
+
+function getRegistrationPageHeader(page) {
```

Comment:
V rámci konzistence s ostatními metodami vracejícími lokátory bych nedával předponu _RegistrationPage_
_getHeader_ sice zní taky divně, ale třeba bychom přišli na lepší název
```suggestion
function getHeader(page) {
```